rh7/SPECS/config

```diff
@@ -1,3 +1,4 @@
 %defattr (-,root,root)
 /lib/modules/%2-%1
 %config /etc/depmod.d/hyperv.conf
+%config /etc/modprobe.d/hyperv_bl_drv.conf
```

Comment:
Why add here and remove in buildrpms.sh?
Just add in buildrpms.sh and nothing to do here.

Reply:
Both buildrpms script and rpmbuild command will work directly.

Reply:
I see there will be two of these lines in config file in case of RH73, 74 & 75. Is this ok?

Reply:
There will be only one line %config /etc/modprobe.d/hyperv_bl_drv.conf.  In case of buildrpms.sh it will be created and if using rpmbuild then static file will be used.